docs/LogGuide.md

```diff
@@ -3042,24 +3050,24 @@ towards.
 If there is some sort of error related to parsing this data from the network packet,
 `dataFlag` will be `256`, and the `x`/`y`/`z`/`heading` fields will be blank.
 
-`globalEffectCounter` is equivalent to `sequence` field in
-[NetworkAbility](#line-21-0x15-networkability) and
-[NetworkAOEAbility](#line-22-0x16-networkaoeability).
-
 Note that unlike [StartsUsingExtra](#line-263-0x107-startsusingextra), you do not need
 to worry about whether or not there is an actor target, as this represents the final
 snapshotted location of the Ability.
 
+Finally, the last field represents the animation lock of the skill in seconds. This is not always meaningful -
+for example, actions with a cast time may report an animation lock of 0.1 seconds. Auto-attacks may also report
+a non-zero cast time, despite the fact that they do not prevent your character from performing actions.
+
 <!-- AUTO-GENERATED-CONTENT:START (logLines:type=AbilityExtra&lang=en-US) -->
 
 #### Structure
 
 ```log
 Network Log Line Structure:
-264|[timestamp]|[sourceId]|[id]|[globalEffectCounter]|[dataFlag]|[x]|[y]|[z]|[heading]
+264|[timestamp]|[sourceId]|[id]|[globalEffectCounter]|[dataFlag]|[x]|[y]|[z]|[heading]|[animationLock]
```

Comment:
It looks like you may have manually added these?  If so, they'll get blown away the next time someone runs `util/gen_log_guide.ts`.  If you run that (`npm run generate-log-guide`), it will automataically add these and also correctly add them to the `Regexes` section below.

Also need to update the example log lines in `util/example_log_lines.ts`, and ideally (although it won't throw a compile error if you don't) add the animationLock property in `test/helper/example_log_lines_test_data.ts`.

Reply:
No, I added them to the netlog_defs, and then ran that script. I'll add it to example_log_lines as well.